homeassistant/components/glances/sensor.py

```diff
@@ -186,3 +201,68 @@ def _schedule_immediate_update(self):
                         self._state = round(mem_use / 1024 ** 2, 1)
                 except KeyError:
                     self._state = STATE_UNAVAILABLE
+
+
+class GlancesDockerSensor(Entity):
+    """Implementation of Glances Docker sensor."""
+
+    def __init__(self, glances_data, name, container_name):
+        """Initialize the sensor."""
+        self.glances_data = glances_data
+        self._name = name
+        self._available = False
+        self.container_name = container_name
+        self.container_info = None
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return "{} Docker {}".format(self._name, self.container_name)
+
+    @property
+    def icon(self):
+        """Icon to use in the frontend, if any."""
+        return ICON_DOCKER
+
+    @property
+    def available(self):
+        """Could the device be accessed during the last update call."""
+        return self.glances_data.available and self._available
+
+    @property
+    def state(self):
+        """Return the state of the resources."""
+        return self.container_info["Status"]
+
+    @property
+    def state_attributes(self):
+        """Return the device state attributes."""
+        attr = {ATTR_DOCKER_ID: self.container_info["Id"]}
+
+        if len(self.container_info["Image"]) > 0:
+            attr[ATTR_DOCKER_IMAGE] = self.container_info["Image"][0]
+
+        if self.container_info["Command"] is not None:
+            attr[ATTR_DOCKER_COMMAND] = self.container_info["Command"]
+
+        if self.container_info["cpu_percent"] is not None:
+            attr[ATTR_DOCKER_CPU_PERCENT] = round(self.container_info["cpu_percent"], 2)
```

Comment:
We prefer to make one sensor entity per measurement.

Reply:
So should there be a seperate sensor for each attribute or just for CPU and CPU Percentage?

Reply:
Maybe just CPU and CPU percentage? Are the other ones useful on their own?